File: solutions/1431. Kids With the Greatest Number of Candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        vector<bool> v;
        int maxx=INT_MIN;
        for(int i=0;i<candies.size();i++){
            maxx=max(maxx,candies[i]);
        }
        for(int i=0;i<candies.size();i++){
            if(candies[i]+extraCandies>=maxx){
                v.push_back(true);
            }
            else v.push_back(false);
        }
        return v;
    }
};

File: solutions/1431. Kids With the Greatest Number of Candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        maxi = max(candies)
        ans = []
        for c in candies:
            if c+extraCandies >= maxi:
                ans.append(True)
            else:
                ans.append(False)
        return ans
